File: src/main/java/com/baulsupp/oksocial/services/instagram/InstagramUtil.java
package com.baulsupp.oksocial.services.instagram;

import com.google.common.collect.Sets;
import java.util.Collections;
import java.util.Set;
import okhttp3.Request;

public class InstagramUtil {
  private InstagramUtil() {
  }

  public static final Set<String> API_HOSTS =
      Collections.unmodifiableSet(Sets.newHashSet(
          "api.instagram.com")
      );

  public static Request apiRequest(String s, Request.Builder requestBuilder) {
    return requestBuilder.url("https://api.instagram.com" + s).build();
  }
}

File: src/main/kotlin/com/baulsupp/oksocial/services/instagram/InstagramUtil.kt
package com.baulsupp.oksocial.services.instagram

import okhttp3.Request

object InstagramUtil {

    val API_HOSTS = setOf((
            "api.instagram.com")
    )

    fun apiRequest(s: String, requestBuilder: Request.Builder): Request {
        return requestBuilder.url("https://api.instagram.com" + s).build()
    }
}
